client_with_ssl.go

```diff
@@ -159,18 +166,24 @@ func fetchCities(client *hazelcast.Client) {
 			fmt.Errorf("iterating: %w", err)
 		}
 
-		id := mustGet(row.GetByColumnName("__key"))
-		city := mustGet(row.GetByColumnName("this")).(City)
+		id, err := row.GetByColumnName("__key")
+		if err != nil {
+			panic(err)
+		}
 
-		fmt.Printf("| %4s | %20s | %20s | %15d |", id, city.Country, city.CityName, city.Population)
-	}
-}
+		c, err := row.GetByColumnName("this")
+		if err != nil {
+			panic(err)
```

Comment:
Fixed at https://github.com/hazelcast/hazelcast-cloud-go-sample-client/pull/14/commits/c0b65931711e21155c5871a3660f9e3783f78825. Btw, I didn't apply the suggestion since closing parentheses is missing.